srcpkgs/pantheon-dpms-helper/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pantheon-dpms-helper'
+pkgname=pantheon-dpms-helper
+version=1.0
+revision=1
+wrksrc="dpms-helper-${version}"
+build_style=meson
+hostmakedepends="cmake pkg-config glib-devel libgee08-devel desktop-file-utils"
```

Comment:
cmake should not be here, and libgee08-devel should not be in hostmakedepends. Likewise I doubt you need both desktop-file-utils dependencies. The hostmakedepends should be enough.